source/rust_verify_test/tests/atomics.rs

```diff
@@ -1,344 +1,380 @@
-#![feature(rustc_private)]
```

Comment:
Same here. It looks like this file was rewritten? Is it again perhaps different newlines?

Reply:
Fixed.